.rubocop.yml

```diff
@@ -10,5 +10,8 @@ Metrics/PerceivedComplexity:
 Metrics/MethodLength:
   Max: 20
 
+Metrics/LineLength:
+  Max: 150
```

Comment:
> The question to ask is, can the line in question be shortened without drastically decreasing readability?

Usually yes, often no. 150 is just as arbitrary as 80. If I had it my way we would not limit line length at all, and depend on the good judgement of our team and hypothetical future contributors to keep the code readable.

[This line](https://github.com/appfolio/rentlinx_client/pull/6/files#diff-190506dce5af57319415f8ab5dea5024R22) is the one that made us make the change. How would you break that under 80 characters? You would have to break it into three lines, and it would make no sense. Or maybe we could rename `remaining_attrs` to `ra` or something? At this point we're writing silly code to satisfy tools, which is a bad road to go down.